src/lib/types/api-user.ts

```diff
@@ -45,7 +53,7 @@ export default class ApiUser {
         if (projects && projects.length > 0) {
             this.projects = projects;
         } else {
-            this.projects = [project];
+            this.projects = project ? [project] : [];
```

Comment:
This is slightly better, previously we were returning a list with a single undefined element